ext/spl/spl_dllist.stub.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+class SplQueue extends SplDoublyLinkedList
+{
+    /**
+     * @param mixed $value
+     * @return bool
+     */
+    public function enqueue($value) {}
+
+    /** @return mixed */
+    public function dequeue() {}
+}
+
+class SplDoublyLinkedList implements Iterator, Countable, ArrayAccess, Serializable
+{
+    /**
+     * @param mixed $index
+     * @param mixed $value
+     * @return void
+     */
+    public function add($index, $value) {}
+
+    /** @return mixed */
+    public function pop() {}
+
+    /** @return mixed */
+    public function shift() {}
+
+    /**
+     * @param mixed $value
+     * @return bool
+     */
+    public function push($value) {}
+
+    /**
+     * @param mixed $value
+     * @return void
+     */
+    public function unshift($value) {}
+
+    /** @return mixed */
+    public function top() {}
+
+    /** @return mixed */
+    public function bottom() {}
+
+    /** @return int */
+    public function count() {}
+
+    /** @return bool */
+    public function isEmpty() {}
+
+    /** @return void */
+    public function setIteratorMode(int $mode) {}
+
+    /** @return int */
+    public function getIteratorMode() {}
+
+    /**
+     * @param mixed $index
```

Comment:
```suggestion
     * @param int $index
```
Maybe? The value gets cast to long, so I think we can provide this hint.